web/app.go

```diff
@@ -99,11 +100,12 @@ func DefaultDependencies(config *Config) Dependencies {
 	sapSystemsService := services.NewSAPSystemsService(consulClient)
 	clustersService := services.NewClustersService(db, checksService)
 	collectorService := services.NewCollectorService(db, projectorWorkersPool.GetChannel())
+	intallationService := services.NewInstallerService(db)
 
 	return Dependencies{
 		consulClient, webEngine, collectorEngine, store, projectorWorkersPool,
 		checksService, subscriptionsService, hostsService, sapSystemsService, tagsService,
-		collectorService, clustersService, hostsServiceNext,
+		collectorService, clustersService, hostsServiceNext, intallationService,
```

Comment:
I see what you mean.

The alternative would be replacing in `web/app.go` this
```
if err := deps.installationService.InitializeIdentifier(); err != nil {
	log.Errorf("failed to initialize installation identifier: %s", err)
	return nil, err
}
```

with this
```
db, err := db.InitDB(config.DBConfig)
if err != nil {
	log.Fatalf("failed to connect database: %s", err)
}

installer := services.NewInstallerService(db)

if err := installer.InitializeIdentifier(); err != nil {
	log.Errorf("failed to initialize installation identifier: %s", err)
	return nil, err
}
```

and so duplicating the db initialization which is already happening in `DefaultDependencies`
```
db, err := db.InitDB(config.DBConfig)
if err != nil {
	log.Fatalf("failed to connect database: %s", err)
}
```

It might be not used elsewhere, yet I see it as a dependency for the App instantiation itself when we call `NewAppWithDeps`.

Any other option?

Reply:
Ok, let's go for it them!